docs/Using Randomness/example-contracts.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: nodes.liquid
+section: ethereum
+date: Last Modified
+title: "Example Contracts"
+permalink: "docs/chainlink-vrf/example-contracts/"
+whatsnext: {"Contract Addresses":"/docs/vrf-contracts/"}
+metadata:
+  description: "Example contracts for generating a random number inside a smart contract using Chainlink VRF."
+  image:
+    0: "/files/OpenGraph_V3.png"
+---
+
+> ℹ️ You are viewing the VRF v2 guide.
+>
+> If you are using v1, see the [VRF v1 guide](/docs/get-a-random-number/v1/).
+
+How you manage the subscription depends on your randomness needs. You can configure your subscriptions using the [Subscription Manager](https://vrf.chain.link) application, but these examples demonstrate how to create your subscription and register your applications programmatically. For these examples, the contract owns and manages the subscription. You can still view the subscriptions in the [Subscription Manager](https://vrf.chain.link) and any wallet can provide funding to those subscriptions.
+
+**Table of contents**
+
+- [Modifying subscriptions and configurations](#modifying-subscriptions-and-configurations)
```

Comment:
Generally speaking, is this the casing we'll be using for subtitles? I've been titling them using MLA title casing so just want to confirm

Reply:
I'm not too particular about heading case, but you are right we should be consistent. This is the only subtitle style I've ever used in software documentation.

- https://cloud.google.com/compute/docs/instances/nested-virtualization/creating-nested-vms
- https://www.ibm.com/docs/en/SSAUZ9_12.1.0/adbugl14.pdf
-